.github/workflows/starknet-rpc-tests.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           ./target/release/madara setup --chain=dev --from-local=configs
       - name: Run rpc test
         run: |-
-          ./target/release/madara --dev --sealing=manual &
+          ./target/release/madara --dev --sealing=manual --da-layer=ethereum --da-conf=examples/da-confs/ethereum.json &
```

Comment:
So we need to run the node with `--da-layer=ethereum --da-conf=examples/da-confs/ethereum.json` in order to be able to answer `getStateUpdate`. What if we don't?

Reply:
the `commitment-state-diff` worker is only initialized if we specify a da layer yes

Reply:
So we are not able to correctly answer the rpc.
Two solutions:
- always spawn `commitment-state-diff` worker and store them in db
- if state-diff-not in db, simulate the block in the runtime and get the state diff out of this 

@apoorvsadana @antiyro @EvolveArt wdyt?

Reply:
see https://github.com/keep-starknet-strange/madara/issues/1428